LINUXCOMPILE.md

```diff
@@ -3,7 +3,7 @@
 ### GNU Compiler
 ```
     # Ubuntu / Debian
-    sudo apt-get install libmicrohttpd-dev libssl-dev cmake build-essential libhwloc-dev
+    sudo apt install libmicrohttpd-dev libssl-dev cmake build-essential libhwloc-dev
```

Comment:
No problem... ```apt``` and ```apt-get``` are different pieces of software (but doing almost the same things)...

```
# ls -la `which apt`
-rwxr-xr-x 1 root root 14376 Mar  8 13:48 /usr/bin/apt

# ls -la `which apt-get`
-rwxr-xr-x 1 root root 43128 Mar  8 13:48 /usr/bin/apt-get
```